lib/ex338_web/coherence_web.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Ex338Web.Coherence.Web do
+  @moduledoc false
+
+  def view do
+    quote do
+      use Phoenix.View, root: "lib/ex338_web/templates",
+                        namespace: Ex338Web
+      # Import convenience functions from controllers
+
+      import Phoenix.Controller, only: [get_csrf_token: 0, get_flash: 2, view_module: 1]
+
+      # Use all HTML functionality (forms, tags, etc)
+      use Phoenix.HTML
+
+      import Ex338Web.Router.Helpers
+      import Ex338Web.ErrorHelpers
+      import Ex338Web.Gettext
+      import Ex338Web.Coherence.ViewHelpers
+    end
+  end
+
+  def controller do
+    quote do
+      use Phoenix.Controller, namespace: Ex338Web, except: [layout_view: 2]
+      use Coherence.Config
+      use Timex
+
+      import Ecto
+      import Ecto.Query
+      import Plug.Conn
+      import Ex338Web.Router.Helpers
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives